src/context/online.rs

```diff
@@ -30,18 +30,26 @@ pub enum AudioContextLatencyCategory {
     Specific(f64),
 }
 
-/// Specify the playback configuration
-/// in non web context, it is the only way to specify
-/// the system configuration
-#[derive(Clone, Debug)]
+impl Default for AudioContextLatencyCategory {
+    fn default() -> Self {
+        Self::Interactive
+    }
+}
+
+/// Specify the playback configuration for the [`AudioContext`] constructor.
+///
+/// All fields are optional and will default to the value best suited for interactive playback on
+/// your hardware configuration.
+///
+/// Check the documentation of the [`AudioContext` constructor](AudioContext::new) for usage
+/// instructions.
+#[derive(Clone, Debug, Default)]
 pub struct AudioContextOptions {
     /// Identify the type of playback, which affects
     /// tradeoffs between audio output latency and power consumption
     pub latency_hint: Option<AudioContextLatencyCategory>,
```

Comment:
Do we need the `Option` here, as it defaults to `interactive`? maybe that would be more consistent to with the rules we followed for the nodes https://github.com/orottier/web-audio-api-rs/pull/103

```
(AudioContextLatencyCategory or double) latencyHint = "interactive";
```

```rs
impl Default for AudioContextOptions {
  fn default() -> Self {
     Self {
       latency_hint: AudioContextLatencyCategory::Interactive,
       sample_rate: None,
     }
  }
}
```

(Actually we probably don't even need to implement as it already seems to be the default)